cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -882,5 +882,6 @@
     public static final String EXEC_THREADS = "metadata.service.exec.threads";
     public static final String METADATA_HANDLERS_NAME = "metadata.handlers";
     public static final String METADATA_UPDATES_KAFKA_TOPIC = "metadata.updates.kafka.topic";
+    public static final String METADATA_UPDATES_PUBLISH_ENABLED = "metadata.updates.publish.enabled";
```

Comment:
Be consistent with the naming. You have `EXEC_THREADS` above, but then you also have `METADATA_` prefixed named. I think the variable name doesn't need to have the `METADATA_` prefix since it's already defined inside `class Metadata`.